### PR TITLE
Upload captured photo ios

### PR DESCRIPTION
**Pull request by SergioEstevao**
Refs #471 

This PR hooks the methods needed to get the device camera capture of media.

### CODE REVIEW

File: react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.swift [hunk 1]
```diff
@@ -38,6 +38,19 @@ public class RNReactNativeGutenbergBridge: RCTEventEmitter {
...
 
+    @objc
+    func onCapturePhotoPressed(_ callback: @escaping RCTResponseSenderBlock) {
+        DispatchQueue.main.async {
+            self.delegate?.gutenbergDidRequestMediaFromCameraPicker(with: { (mediaID, url) in
+                guard let url = url, let mediaID = mediaID else {
+                    callback(nil)
+                    return
+                }
+                callback([mediaID, url])
```
Comment (etoledom) [inline react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.swift]: Just an idea:
All the media events in the bridge look super similar. (`onMediaLibraryPressed`. `onUploadMediaPressed`, `onCapturePhotoPressed`)
Maybe we could merge them and send a flag to identify them?

Comment (SergioEstevao) [reply react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.swift]: Just proposed that on the Android PR that has the bridge method for JS. I can also change the delegate here to make it a single method.

Comment (SergioEstevao) [reply react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.swift]: Here: https://github.com/WordPress/gutenberg/pull/13450#pullrequestreview-196011991

Comment (etoledom) [reply react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.swift]: Heh nice! 👍

**Pull request merged**